develop/docs/dlq.md

```diff
@@ -0,0 +1,28 @@
+# Managing DLQ Using tdbg
+When a task in the Temporal server encounters a terminal error, it may be enqueued in the Dead Letter Queue (DLQ). 
+To effectively manage these DLQ messages, follow these steps:
+
+1. Search `Task enqueued to DLQ` in logs and see task details.
+2. From the log trace, you can identify the offending namespace, task type, source cluster, and target cluster of the task.
+3. From the task type, get the corresponding dlq type from this map:
+    1. transfer: 1
+    2. timer: 2
+    3. replication: 3
+    4. visibility 4
+4. Search `Marking task as terminally failed, will send to DLQ` in logs and find the terminal error that caused the task to be enqueued to DLQ.
+5. You can list the DLQ messages using
+   `tdbg dlq --dlq-version v2 read --dlq-type {type}`. Substitute `{type}` with the integer value from step 3.
```

Comment:
How do they know what the message ID will be from the logs?

Reply:
Added message id in DLQWriter log and mentioned that in doc.